hbase10/README.md

```diff
@@ -49,39 +49,36 @@ hbase(main):002:0> create 'usertable', 'cf', {SPLITS => (1..n_splits).map {|i| "
 
 Make a note of the column family, in this example it's `cf``.
 
-### 4. Fetch the Proper ALPN Boot Jar
+### 4. Download the Bigtable Client Jar with required dependencies:
```

Comment:
Now that there's a `googlebigtable` module, do folks who want to use YCSB + Bigtable need to use `hbase10` module for testing or can they just use `googlebigtable`?

In other words, are these instructions an optional alternative or are they still required?

The instructions in https://github.com/brianfrankcooper/YCSB/tree/master/googlebigtable appear to not depend on `hbase10` so can we just remove the Bigtable instructions from HBase-related modules for clarity?

Reply:
googlebigtable tests the core (proto) client. The instructions in hbase10 are for testing the bigtable hbase api